File: unity/Assets/Hallway/Constructor/RPG/firstVisitLecture.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.firstVisitLecture) {
		scr1.Achievement("Первое посещение лекционного зала!\n+10 очков!", 10);
		scr1.RPG.firstVisitLecture = true;
	}
}

File: unity/Assets/Hallway/Constructor/RPG/firstVisitLecture.cs
using UnityEngine;
using System.Collections;

public class firstVisitLecture : MonoBehaviour
{

    GameObject Bootstrap;

    void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.firstVisitLecture)
        {
            rpgParser.Achievement("Первое посещение лекционного зала!\n+10 очков!", 10);
            rpgParser.RPG.firstVisitLecture = true;
        }
    }
}
